Uchu.StandardScripts/General/DeathPlane.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Numerics;
+using System.Threading.Tasks;
+using InfectedRose.Core;
+using InfectedRose.Lvl;
+using Uchu.Physics;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.General
+{
+    /// <summary>
+    /// Native implementation of scripts/ai/act/l_act_player_death_trigger.lua
+    /// </summary>
+    [ScriptName("l_act_player_death_trigger.lua")]
+    public class DeathPlane : ObjectScript
+    {
+        /// <summary>
+        /// Creates the object script.
+        /// </summary>
+        /// <param name="gameObject">Game object to control with the script.</param>
+        public DeathPlane(GameObject gameObject) : base(gameObject)
+        {
+            //Console.WriteLine("Death barrier positioned at " + gameObject.Transform.Position + ", Scale of GameObject is " + gameObject.Transform.Scale);
+
+            var physics = gameObject.GetComponent<PhysicsComponent>();
+            if (physics == null || physics == default) return;
```

Comment:
This check is the same. `default` is used to check if a value type is default in memory (`0`, `false`, etc) or is null, or just if an object type is null. Since the component is an object, this becomes `if (physics == null || physics == null) return;`.

Reply:
Will fix.